devicetypes/smartthings/aeotec-doorbell-siren-child.src/aeotec-doorbell-siren-child.groovy

```diff
@@ -26,12 +26,15 @@ metadata {
 				attributeState "chime", label: 'off', action: 'chime.off', icon: "st.alarm.alarm.alarm", backgroundColor: "#ff0000"
 			}
 		}
+		standardTile("off", "device.alarm", inactiveLabel: false, decoration: "flat") {
```

Comment:
Is the purpose of this to provide an "off" command? If so, you already have the one for chime off, which should operate the same command. You might just want to change this to a value tile (and also rename it to "alarm" as you currently have two "off" tiles), or remove it altogether, as both chime and alarm events are sent to the child device.